### ISSUE TITLE
Displaying events of day clicked on

### ISSUE THREAD

**Issue by sebmins** [SundeepK/CompactCalendarView#169]
I am very new to android only been using since September. I'm trying to make it so when you add an event to a specific day then click on that day it will display the "data" part of the event.

I had created an event 
`final Event ev1 = new Event(Color.GREEN, 1488110400000L, footballEv + time12);`
` List<Event> events = compactCalendarView.getEvents(dateClicked);`

And when trying to get it to display I have used three different methods
I have tried ev1.getData() this will only work to retrieve that specific event it isnt affected by clicking on different days.

I have tried just displaying "events" this again works but displays the whole colour, time and data.
I also tried events.get(1), this has the same problem as before.

I think this should be such an easy problem to fix I posted it here, am I missing anything?

Any help is appreciated,
Many thanks
Seb

**Comment by SundeepK** [comment]
Have a look at the sample app provided. It adds some events using the `onCreate` method in the activity. Then when you click the day you want to select, it gets the events for the day and displays in a listview. 

Is this what you want? Remember that: `List<Event> events = compactCalendarView.getEvents(dateClicked);` _returns_ a `List` object. You must iterate through the list and call `getData()` on each and every event to get the data.

**Comment by sebmins** [comment]
Yeah I have looked at the sample, I created the event in the onCreate method.


What do you mean iterate through the list and call getData()? Because if I try:
`events.getData()` 
It won't work

**Comment by SundeepK** [comment]
```java
List<Event> events = compactCalendarView.getEvents(dateClicked);
for (Event event : events ) {
    Log.d("myclass", " Event data : " + event.getData());
}
```

**Comment by sebmins** [comment]
Thanks man! You are so helpful :) keep up the good work

**Comment by sebmins** [comment]
Since last time I have been trying to display my events in a `TextView`, I used a `StringBuilder` and appended the events for each loop `stringBuilder.append(eventToString + "\n");` and set the text after the loop. This works but a new line isn't printed out. I tried using google and `"\n"` doesn't work in android. 

I have also tried using an adapter and a `ListView` but this doesnt work either, I was wondering has anyone else got this to work?

I'm also trying to display the time of the event on the same line, I created another `TextView` and convert the time in milliseconds to hours and minutes, this works but I again cannot print them on a new line.

Many thanks,
Seb
